sample.txt

```diff
@@ -1,2 +1,3 @@
 hey
 matthew
+hey!
```

Comment:
testing...incoming communication...I have uploaded my current tutorial project from Android Studio to GitHub (MyFirstApp2-2). Ignore 'MyFirstApp2", as this is empty.  My previous issue has been resolved.  I am now at snag#2.  Everything looks good when running the app, but when I attempt to enter a message in the empty text box on the home screen (when i tap the 'Enter a message' to enter a message) the app automatically switches from the home screen to a secondary screen which is intended to display the entered text AFTER pressing send and thus I am unable to enter said text and thereby display it on secondary screen.  The tutorial I am following can be found here:https://developer.android.com/training/basics/firstapp